tests/scancode/data/long_file_name/expected.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
You should not care about the detailed scan results when you tests name shortening in the CLI ui.